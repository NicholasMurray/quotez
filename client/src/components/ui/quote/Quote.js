const Quote = ({ quote }) => {
  return (
    <>
      <h2>"{quote.en}"</h2>
      <h3>
        <i>{quote.author}</i>
      </h3>
    </>
  );
};

export default Quote;
