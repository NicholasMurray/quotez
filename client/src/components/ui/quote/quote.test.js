import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';
import Quote from '.';

const quote = {
  en: 'Hello World',
  author: 'A.N. Other',
};

test('it renders the quote correctly', () => {
  const tree = TestRenderer.create(<Quote quote={quote} />).toJSON();
  expect(tree).toMatchSnapshot();
});
