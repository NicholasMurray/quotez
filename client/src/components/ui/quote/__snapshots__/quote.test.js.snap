// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the quote correctly 1`] = `
Array [
  <h2>
    "
    Hello World
    "
  </h2>,
  <h3>
    <i>
      A.N. Other
    </i>
  </h3>,
]
`;
